Hackney.Shared.Tenure/Domain/TenureInformation.cs

```diff
@@ -31,5 +31,18 @@ public class TenureInformation
         public bool? IsSublet { get; set; }
         public DateTime? SubletEndDate { get; set; }
         public int? VersionNumber { get; set; }
+        public string JigsawRef { get; set; }
+        public string FundingSource { get; set; }
+        public int? NoOfAdults { get; set; }
+        public int? NoOfChildren { get; set; }
+        public bool? HasStorage { get; set; }
+        public bool? RentAccountRequired { get; set; }
+        public string NoRentAccountReason { get; set; }
+        public DateTime? ActualStartOfTenure { get; set; }
+        public DateTime? RentLetterDate { get; set; }
+        public DateTime? RentCardDate { get; set; }
```

Comment:
All of the Rent account related information should instead be in an 'account' object and should contain the various data properties related to the rent account, possibly including the payment reference (which would require a new API endpoint version).